src/frontends/common/include/openvino/frontend/frontend.hpp

```diff
@@ -144,6 +144,11 @@ class FRONTEND_API FrontEnd {
     }
 
 protected:
+    /// \brief Get frontend library pointer.
+    ///
+    /// \return const std::shared_ptr<void>& to shared library pointer.
+    const std::shared_ptr<void>& get_library() const;
```

Comment:
Why do we need this method? Why objects what are using this method cannot work with the Frontend class which holds shared library?